sdk/monitor/azure-monitor-query/azure/monitor/query/_logs_query_client.py

```diff
@@ -43,20 +44,22 @@ class LogsQueryClient(object): # pylint: disable=client-accepts-api-version-keyw
 
     :param credential: The credential to authenticate the client.
     :type credential: ~azure.core.credentials.TokenCredential
-    :keyword endpoint: The endpoint to connect to. Defaults to 'https://api.loganalytics.io'.
+    :keyword endpoint: The endpoint to connect to. Defaults to 'https://api.loganalytics.io/v1'.
     :paramtype endpoint: Optional[str]
     """
 
     def __init__(self, credential: TokenCredential, **kwargs: Any) -> None:
-        endpoint = kwargs.pop("endpoint", "https://api.loganalytics.io")
+        endpoint = kwargs.pop("endpoint", "https://api.loganalytics.io/v1")
```

Comment:
Not for this PR, but should we accept an `api_version` keyword only argument in the client?

Reply:
Yea, I think this would make sense for MetricsQueryClient since the underlying operations include `api_version` in the requests. No such thing for LogsQueryClient, however, since API versioning is based on what endpoint the user submits a request to.